tracer/src/Datadog.Trace/Ci/Agent/MessagePack/EventMessagePackFormatter.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="EventMessagePackFormatter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Vendors.MessagePack;
+using Datadog.Trace.Vendors.MessagePack.Formatters;
+
+namespace Datadog.Trace.Ci.Agent.MessagePack
+{
+    internal abstract class EventMessagePackFormatter<T> : IMessagePackFormatter<T>
+    {
+        private readonly IDatadogLogger _log;
+        // .
```

Comment:
What's with these comments? 🤨

Reply:
removed in [52254ed](https://github.com/DataDog/dd-trace-dotnet/pull/2402/commits/52254ed72ce5138672ff33cbffe51ac87d3d1d2e)